tools/ci/src/main.rs

```diff
@@ -1,19 +1,47 @@
-use xshell::cmd;
+use std::io::BufRead;
+
+use duct::cmd;
 
 fn main() {
     // When run locally, results may from actual CI runs triggered by .github/workflows/ci.yml
     // - Official CI runs latest stable
     // - Local runs use whatever the default Rust is locally
 
     // See if any code needs to be formatted
-    cmd!("cargo fmt --all -- --check")
+    println!("$ cargo fmt --all -- --check");
+    let errput = cmd!("cargo", "fmt", "--all", "--", "--check")
```

Comment:
Would it be possible, to only use the check flag, when this is run as Part of CI?
It would be more ergonomic, for Users if they don't have to fix it manually.

Reply:
I don't want to do that _in this PR._ A design goal of mine is that the command `cargo ci` _by itself_ act as identically as possible across all platforms and versions of Rust.

I'm certainly open to the ideas of either adding another alias (`cargo ci-local`?) or an argument (`cargo ci --auto`?) to get that changed behavior.  First, I'd like to see that someone actually wants it (maybe you?).  Then I think it would be best to implement that in it's own pull request. If that's something you feel like taking on, I'd be happy to be your reviewer and/or collaborator. 👍🏼